docs/configuration.md

```diff
@@ -1188,7 +1188,9 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.task.maxFailures</code></td>
   <td>4</td>
   <td>
-    Number of individual task failures before giving up on the job.
+    Number of failures of any particular task before giving up on the job. The same
+    task has to fail this number of attempts, it isn't affected by the number of failures
```

Comment:
I can see the 2 sentence not being clear, but the second part of your suggestion about "if one particular task has failed" sounds a bit out of place to me, how about:

"The total number of failures spread across different tasks will not cause the job to fail, it has to be a particular task failing this number of attempts."